AsyncDisplayKit/ASCollectionNode.mm

```diff
@@ -217,6 +218,12 @@ - (_ASCollectionPendingState *)pendingState
   return _pendingState;
 }
 
+- (void)setInverted:(BOOL)inverted
+{
+  self.view.dataController.inverted = inverted;
+  self.transform = inverted ? CATransform3DMakeRotation(M_PI, 0, 0, 1.0) : CATransform3DIdentity;
```

Comment:
Let's use `CATransform3DMakeScale(1, -1, 1)` instead of rotation. It's cleaner because π can't be represented perfectly.